modulefiles/module_base.orion.lua

```diff
@@ -45,10 +43,14 @@ setenv("WGRIB","wgrib")
 setenv("UTILROOT",(os.getenv("prod_util_ROOT") or "None"))
 
 --prepend_path("MODULEPATH", pathJoin("/work/noaa/global/glopara/git/prepobs/v" .. (os.getenv("prepobs_run_ver") or "None"), "modulefiles"))
-prepend_path("MODULEPATH", pathJoin("/work/noaa/global/glopara/git/prepobs/feature-GFSv17_com_reorg_log_update/modulefiles"))
-load(pathJoin("prepobs", (os.getenv("prepobs_run_ver") or "None")))
-
-prepend_path("MODULEPATH", pathJoin("/work/noaa/global/glopara/git/Fit2Obs/v" .. (os.getenv("fit2obs_ver") or "None"), "modulefiles"))
-load(pathJoin("fit2obs", (os.getenv("fit2obs_ver") or "None")))
+--prepend_path("MODULEPATH", "/work/noaa/global/glopara/git/prepobs/feature-GFSv17_com_reorg_log_update/modulefiles")
+--load(pathJoin("prepobs", (os.getenv("prepobs_run_ver") or "None")))
```

Comment:
I'm good with these staying as comments temporarily since the version will presumably be updated later on all systems, at which time they will be restored.

Reply:
Thanks!